src/Database/index.ts

```diff
@@ -1,48 +1,52 @@
-// @flow
-import { type Predicate, sample, first, last, filter } from "lodash";
+import * as sample from 'lodash.sample';
+import * as first from 'lodash.first';
+import * as last from 'lodash.last';
+import * as filter from 'lodash.filter';
 
 const databaseCollectionStores: WeakMap<
   Database<any>,
-  CollectionStore<Object>
+  CollectionStore<any>
 > = new WeakMap();
 
-export type DatabaseSchema = {
-  +[collectionName: string]: Object
-};
+export interface DatabaseSchema {
+  [collectionName: string]: any;
```

Comment:
why not keep Object?

Reply:
Well, when you put it an Object it mess some shit up.